readthedocs/builds/models.py

```diff
@@ -158,9 +162,22 @@ def vcs_url(self):
         """
         Generate VCS (github, gitlab, bitbucket) URL for this version.
 
-        Branch/Tag Example: https://github.com/rtfd/readthedocs.org/tree/3.4.2/.
-        Pull/merge Request Example: https://github.com/rtfd/readthedocs.org/pull/9999/.
+        Example: https://github.com/rtfd/readthedocs.org/tree/3.4.2/.
+        External Version Example: https://github.com/rtfd/readthedocs.org/pull/99/commits/b630b630/.
```

Comment:
Don't we want this URL to point to the PR still? Otherwise we are changing the meaning of this function based on whether the version is external or internal.

We only want specific commits to link to a commit, not the version itself. We need another function to add a specific URL to the commit I think.